corehq/apps/tour/templates/tour/config/new_blank_app.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
+<script type="text/javascript">
```

Comment:
Also, I would wait to see how we actually use this feature IRL before we even think of abstracting it further.